yake_rust/Cargo.toml

```diff
@@ -14,6 +14,7 @@ repository = "https://github.com/quesurifn/yake-rust"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+serde = ["dep:serde"]
```

Comment:
You can omit it. `serde = { optional = true }` automatically allocates a feature.